CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* None.
+* Added the functionality to delete files for a given SharedRealm, unlocking  ([realm-dotnet#386](https://github.com/realm/realm-dotnet/issues/386)).
```

Comment:
This was missing a reference to this PR or an issue. It would also be good to mention what that function is called. 
cc @ironage @finnschiermer @DominicFrei

Reply:
@bmunkholm why is the linked issue in dotnet insufficient?

Reply:
@ironage That doesn't give a reference to the code change (no one can see how to use this functionality unless they go search in recently closed PR's).

Reply:
The last comment in the GitHub issue links to this PR.

Reply:
Hmm- ok then. I was admittedly not reading that issue as it only said "unlocking" :-). I still think it doesn't hurt to mention the PR number to core here and which function has been added. Think about it from the perspective of another SDK team reading what new functionality has been added.